### ISSUE TITLE
StyledPlayerView: Support skipping of minimal progress bar step as controls timeout and disappear

### ISSUE THREAD

**Issue by yoobi** [google/ExoPlayer#10318]
When filing a question:
-------------------------

I've noticed recently that the `<DefaultTimeBar />` and `<View id="@+id/exo_progress_placeholder" />` have a different show_timeout compared to the rest of the `exo_styled_player_control_view.xml` layout. 
Is there a specific reason ? How can we show/hide everything all together.

When filing a question:
-------------------------

Simply use your own demo app and tap once on a video to show the control, you'll see that the `progress_placeholder` is staying longer that the rest of the layout.

**Comment by ojw28** [comment]
Are you referring to the way the progress bar collapses down to a thin line, and the fact there is then an additional delay before the progress bar disappears entirely? If so, that's the way our UX designers designed it to work, and it doesn't appear like it's possible to customize the behavior.

I will mark this as a low priority enhancement request, but note that our out-of-the-box UI components are, for the most part, intended to be for those who are happy taking a default implementation to avoid having to implement their own. If you have very precise requirements, you should be implementing your own UI components instead.

**Comment by yoobi** [comment]
> Are you referring to the way the progress bar collapses down to a thin line, and the fact there is then an additional delay before the progress bar disappears entirely?

Yes this is what I'm referring to. I've been using the same custom layout since 2 years, it was working as intended but it seems to have changed recently while upgrading version.

Are you saying I shouldn't use `exo_styled_player_control_view.xml` to customize the UI the way I need it ?

**Comment by ojw28** [comment]
We are moving away from officially supporting XML based layout customization, because it didn't work particularly well, and because it was very difficult to keep working across releases.

If you have custom UI requirements you're probably better off creating your own components (optionally by forking the ones that ExoPlayer provides out-of-the-box as a convenient starting point).

**Comment by ArcherEmiya05** [comment]
> Are you referring to the way the progress bar collapses down to a thin line, and the fact there is then an additional delay before the progress bar disappears entirely? If so, that's the way our UX designers designed it to work, and it doesn't appear like it's possible to customize the behavior.
> 
> I will mark this as a low priority enhancement request, but note that our out-of-the-box UI components are, for the most part, intended to be for those who are happy taking a default implementation to avoid having to implement their own. If you have very precise requirements, you should be implementing your own UI components instead.

Good day, we are having trouble where progress bar collapses down in a couple of seconds while playing but a custom controller below it collapse/hide more late, thus ended up with overlapping control and progress. Is there a way to synchronize them? Thanks
![Screenshot_2022-06-28-04-05-21](https://user-images.githubusercontent.com/38008900/176026666-05e9b9cc-8d54-4de2-996c-a49beb8603fd.png)
